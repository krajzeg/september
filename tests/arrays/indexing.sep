array := [1,2,3]
print("Let's print 1 2 3 from the array:", array[0], array[1], array[2])
print("Now, let's do it backwards:", array[-1], array[-2], array[-3])
