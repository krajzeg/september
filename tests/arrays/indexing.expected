Let's print 1 2 3 from the array: 1 2 3
Now, let's do it backwards: 3 2 1
We've modified the array, so it should be 1 42 6 now: 1 42 6
array[end] should also be valid syntax and give 6: 6
